Doc/README.rst

```diff
@@ -0,0 +1,64 @@
+Contributing to the Biopython Tutorial
+======================================
+
+Installing the tools you need
+-----------------------------
+
+On Ubuntu, you can install these with the following commands::
+
+    sudo apt install hevea
+    sudo apt install texlive-latex-base
+    sudo apt-get install texlive-latex-extra
+    sudo apt-get install python-pygments
+
+Formatting code examples
+------------------------
+
+Code examples should be formatted using minted. Also, the Biopython tutorial uses its own system for testing code examples, which is located in ``test_Tutorial.py``. You can read more about this system in `section 21.4 of the Biopython Tutorial <http://biopython.org/DIST/docs/tutorial/Tutorial.html#htoc316>`_.
+
+Here is an example::
+
+    %doctest path-to-folder
+    \begin{minted}{python}
+    from Bio import SeqIO
+    \end{minted}
+
+Formatting text
+---------------
+
+Format headings as follows:
+
+Chapter titles::
+
+    \chapter{Title of Chapter}
+    \label{chapter:chapterlabel}
+
+Section titles::
+
+    \section{Title of Section}
+    \label{sec:sectionlabel}
+
+Subsection titles::
+
+    \section{Title of Subsection}
```

Comment:
Should be ``\subsection{Title of Subsection}``

Are you listing these in order to emphasise the label convention?

Reply:
@peterjc Yes, emphasizing the label convention was the idea. I am not sure if it is really necessary though.